website/docs/r/nsxt_distributed_firewall_rule.html.markdown

```diff
@@ -0,0 +1,117 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_distributed_firewall_rule"
+sidebar_current: "docs-vcd-resource-nsxt-distributed-firewall-rule"
+description: |-
+  The Distributed Firewall allows user to segment organization virtual data center entities, such as
+  virtual machines, based on virtual machine names and attributes. 
+---
+
+# vcd\_nsxt\_distributed\_firewall\_rule
+
+The Distributed Firewall allows user to segment organization virtual data center entities, such as
+virtual machines, based on virtual machine names and attributes. 
+
+~> There is a different resource
```

Comment:
Also, feels like this should be in "red", if possible.

Reply:
<img width="701" alt="image" src="https://github.com/vmware/terraform-provider-vcd/assets/15804230/e1a4f520-d2a3-43a8-b947-fa6adfb93610">